wordpress.org/public_html/wp-content/plugins/plugin-directory/email/class-plugin-approved.php

```diff
@@ -29,32 +29,35 @@ function body() {
 To answer some common questions:
 
 * You must use SVN to upload your code -- we are unable to do that for you
-* Your SVN username is %2$s and your password is the same as you use to log in to WordPress.org
-* Your username is case sensitive
+* Your SVN username is `%2$s` and is case sensitive.
+* Generate your SVN password in your [WordPress.org profile](https://profiles.wordpress.org/%2$s/profile/edit/group/3/?screen=svn-password).
```

Comment:
SVN username/user_login is not the same as the the slug/nicename.

`%4$s` will need to be added, that references `$user->user_nicename`

Reply:
Done in [13db8ab](https://github.com/WordPress/wordpress.org/pull/362/commits/13db8ab5d545874775970016f3a324d5c91c58e1).